pyqtgraph/graphicsItems/ROI.py

```diff
@@ -213,7 +213,7 @@ def angle(self):
         """Return the angle of the ROI in degrees."""
         return self.getState()['angle']
         
-    def setPos(self, pos, update=True, finish=True):
+    def setPos(self, pos, y=None, update=True, finish=True):
```

Comment:
`QGraphicsItem.setPos()` accepts separate x, y arguments. For some reason `ROI.setPos()` did not follow the same pattern and instead only accepted a single argument. Adding `y` here simply recovers the original behavior.